ecs-init/config/common.go

```diff
@@ -117,11 +122,14 @@ func DockerUnixSocket() (string, bool) {
 
 // getBaseLocationForRegion fetches the bucket URI from list of S3 Buckets by region name or default if key is not found
 func getBaseLocationForRegion(regionName string) string {
+	if regionName == RegionNameNotFound {
```

Comment:
That comment is specifically about the tests not covering the case, not whether the underlying code works.  I don't think it matters whether the code separates those two cases, but it does matter that both are covered by tests.  As long as we have test coverage that exercises the different situations we expect to have happen I'll be happy.